common/primitives/src/messages.rs

```diff
@@ -130,6 +139,43 @@ impl<T> BlockPaginationResponse<T> {
 	}
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+/// A trait for helping setup state for running benchmarks.
+pub trait BenchmarkHelper<AccountId> {
+	/// Sets the delegation relationship of between Provider and Delegator.
+	fn set_delegation_relationship(
+		provider: ProviderId,
+		delegator: DelegatorId,
+		schemas: Vec<SchemaId>,
+	) -> DispatchResult;
+
+	/// Sets a publickey to an MSA.
+	fn add_key(msa_id: MessageSourceId, key: AccountId) -> DispatchResult;
+
+	/// Sets the schema count.
+	fn set_schema_count(schema_id: SchemaId);
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+impl<AccountId> BenchmarkHelper<AccountId> for () {
```

Comment:
why do you need this?

Reply:
it makes things easier
```
type Helper = ()
```

Reply:
Maybe we need a comment to describe this is mainly for tests